graphql/schemabuilder/reflect.go

```diff
@@ -612,8 +612,8 @@ func (sb *schemaBuilder) buildStruct(typ reflect.Type) error {
 }
 
 var scalars = map[reflect.Type]string{
-	reflect.TypeOf(bool(false)): "bool",
-	reflect.TypeOf(int(0)):      "int",
+	reflect.TypeOf(bool(false)): "Boolean",
+	reflect.TypeOf(int(0)):      "Int",
```

Comment:
Hm. In, that case, I think I prefer sticking to our Go scalar types: If we expose only some types as GraphQL scalar types, and others as Go scalars, almost any user will still have to deal with Go scalar types. That means the upside of this change is pretty minimal (since users will still have to deal with Go types), while the downside is the inconsistency between the two different naming schemes.